js/microprofile-config-callback.js

```diff
@@ -197,15 +256,54 @@ var microprofileConfigCallBack = (function() {
         }
     };
 
+    var __addVariableToServerXML = function(stepName) {
+        if (stepName === undefined) {
+            stepName = stepContent.getCurrentStepName();
+        }
+        var portVariableElement = '   <variable name="port" value="9084"/>';
+        // reset content every time property is added through the button so as to clear out any manual editing
+        contentManager.focusTabbedEditorByName(stepName, serverXmlFileName);
+        contentManager.resetTabbedEditorContents(stepName, serverXmlFileName);
+        contentManager.replaceTabbedEditorContents(stepName, serverXmlFileName, 10, 10, portVariableElement);
+    };
+
+    var __addVariableToServerXMLButton = function(event, stepName) {
+        if (event.type === "click" ||
+           (event.type === "keypress" && (event.which === 13 || event.which === 32))) {
+            // Click or 'Enter' or 'Space' key event...
+            __addVariableToServerXML(stepName);
+        }
+    };
+
+    var __addappPropertiesToServerXML = function(stepName) {
+        if (stepName === undefined) {
+            stepName = stepContent.getCurrentStepName();
+        }
+        var portappPropertiesPropertyElement = '        <appProperties>\n           <property name="port" value="9085"/>\n        </appProperties>';
```

Comment:
remove two spaces in the string of `'       <appProperties>'` so that it would line up